sql/src/main/java/io/crate/exceptions/RelationUnknownException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.exceptions;
+
+import io.crate.sql.tree.QualifiedName;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.Locale;
+
+public class RelationUnknownException extends ResourceUnknownException {
+
+    private final QualifiedName qualifiedName;
+
+    public RelationUnknownException(String tableName, @Nullable String schemaName) {
+        super(String.format(Locale.ENGLISH, "Cannot resolve relation '%s'", schemaName == null
+            ? new QualifiedName(tableName)
+            : new QualifiedName(Arrays.asList(schemaName, tableName))));
+        if (schemaName == null) {
```

Comment:
Might want to consider taking the QualifiedName in the constructor and in addition have a static factory method for tableName/schemaName.

Usually we also have schemaName first and second table name (See TableIdent). Might want to change it here for consistency but I don't really mind.